elpy/blackutil.py

```diff
@@ -43,7 +51,10 @@ def fix_code(code, directory):
     # Get black config from pyproject.toml
     line_length = black.DEFAULT_LINE_LENGTH
     string_normalization = True
-    pyproject_path = os.path.join(directory, "pyproject.toml")
+    if find_pyproject_toml:
+        pyproject_path = find_pyproject_toml((directory,))
+    else:
+        pyproject_path = os.path.join(directory, "pyproject.toml")
     if toml is not None and os.path.exists(pyproject_path):
```

Comment:
I think #1955 can be fixed by changing this line to:

``` python
if toml and pyproject_path and os.path.exists(pyproject_path):
```

I have hand-tested this, and it seems to cure the issue both in 0803e76e423286c92070eeeca0f2fe05df799b11 and in `master`, so I think it should do the trick.